frontend/plugins/frontline_ui/src/widgets/automations/modules/facebook/components/bots/components/AutomationBotSheetForm.tsx

```
@@ -0,0 +1,246 @@
+      </Sheet.Footer>
+    </>
+  );
+};
+
+export const AutomationBotSheetForm = () => {
+  const [facebookBotId, setFacebookBotId] =
+    useQueryState<string>('facebookBotId');
+  const [isOpen, setOpen] = useState(false);
+
+  useEffect(() => {
+    facebookBotId && setOpen(true);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)